web/views/quote_view.ex

```diff
@@ -1,3 +1,68 @@
 defmodule HonGraffitiPhoenix.QuoteView do
   use HonGraffitiPhoenix.Web, :view
+
+  # This captures(named style) on a caret followed by EITHER one of the characters, or 3 digits
+  @markdown_regex ~r/\^((?<style>(\d{3})|[wrbymnpkotvg*]))/i
```

Comment:
Markdown is technically a specific type of markup, so I think we should use either `markup` or maybe even `hon_markup` instead.